labellab-client/src/components/project/images.js

```diff
@@ -29,7 +17,7 @@ class ImagesIndex extends Component {
     this.state = {
       images: [],
       file: '',
-      image_names: [],
+      image_names: '',
```

Comment:
Doesn't require to be an array. We only use it when there is a single file

Reply:
Revert it back. Unnecessary change. Passing it as an array is intended. Even if there's one image.

Reply:
The reason behind it is that if in the future we decide to pass multiple files, changes will be minimal.